crates/interledger-store-redis/src/account.rs

```diff
@@ -121,7 +124,7 @@ impl ToRedisArgs for Account {
         self.id.write_redis_args(&mut rv);
         if !self.ilp_address.is_empty() {
             "ilp_address".write_redis_args(&mut rv);
-            rv.push(self.ilp_address.to_vec());
+            rv.push(self.ilp_address.to_bytes().to_vec()); // TODO: Can we go directly to Vec<u8>?
```

Comment:
This avoids the `Bytes.clone()` but I'm not sure how much that really makes a difference since we're copying the address bytes anyway
```suggestion
            rv.push((self.ilp_address.as_ref() as &[u8]).to_vec());
```